crates/sncast/src/starknet_commands/account/import.rs

```diff
@@ -196,9 +201,57 @@ fn get_private_key_from_input() -> Result<Felt> {
     parse_input_to_felt(&input)
 }
 
+pub fn get_import_name(
+    import_name: Option<String>,
+    accounts_file: &Utf8PathBuf,
+) -> (String, Option<u32>) {
```

Comment:
1. I think `generate_account_name` could be better.
2. The function should return only one value which is the account name.